Firmware/mmu2.cpp

```diff
@@ -312,6 +312,8 @@ void MMU2::ToolChangeCommon(uint8_t slot){
     while( ! ToolChangeCommonOnce(slot) ){ // while not successfully fed into extruder's PTFE tube
         // failed autoretry, report an error by forcing a "printer" error into the MMU infrastructure - it is a hack to leverage existing code
         // @@TODO theoretically logic layer may not need to be spoiled with the printer error - may be just the manage_response needs it...
+        // overcome blocking conditions
+        lastErrorCode = ErrorCode::OK;
```

Comment:
If this is to fix the LCD blank issue I'm against this change. The LCD blank issue was fixed in https://github.com/prusa3d/Prusa-Firmware/pull/3907

Reply:
yes, this should be removed :+1: